plugins/quetz_sql_authenticator/README.md

```diff
@@ -0,0 +1,35 @@
+## SQL Authenticator
+
+An authenticator that stores credentials in the Quetz SQL database using passlib. Ships with CLI tools for CRUD operations on the credentials table.
+
+### Dependencies
+
+This plugin depends on `passlib`.
```

Comment:
Probably not necessary to mention that here, as we list it as `install_requires` in the `setup.py`.